docker/Dockerfile

```diff
@@ -1,4 +1,11 @@
-FROM openjdk:8u212-jre-alpine
+FROM maven:3.5-jdk-8-alpine as builder
+WORKDIR /usr/src/app
+COPY cdkdepict-lib cdkdepict-lib
+COPY cdkdepict-webapp cdkdepict-webapp
+COPY pom.xml pom.xml
```

Comment:
I forget the syntax but could this not be:
```
COPY cdkdepict-lib cdkdepict-webapp pom.xml /usr/src/app
RUN mvn clean package -f /usr/src/app/pom.xml
```

Perhaps

```
COPY cdkdepict-* pom.xml /usr/src/app
RUN mvn clean package -f /usr/src/app/pom.xml
```
is more future proof?

Reply:
Never mind I forgot Docker COPY doesn't work sanely/as I expect :-).